.all-contributorsrc

```diff
@@ -0,0 +1,65 @@
+{
+  "projectName": "fireloop-starter",
+  "projectOwner": "beeman",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "contributors": [
+    {
+      "login": "brannon-darby",
+      "name": "Brannon N. Darby II",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/6089253?v=3",
+      "profile": "https://github.com/brannon-darby",
+      "contributions": [
```

Comment:
I think for all of us the contributions will be only `code` at this point, as we haven't really done any of the other contributions yet.